app/models/note.rb

```diff
@@ -1,3 +1,5 @@
 class Note < ApplicationRecord
   belongs_to :user
+
+  scope :search_by_title_or_body, ->(substring) { where("title ILIKE ? OR body ILIKE ? ", "%#{substring}%", "%#{substring}%") }
```

Comment:
To avoid the double string interpolation, you could do
```ruby
where("title ILIKE '%?%' OR body ILIKE '%?%'", substring, substring) 
```

Reply:
But this syntax would result in this query
`SELECT  "notes".* FROM "notes" WHERE "notes"."user_id" = $1 AND (title ILIKE '%'new'%' OR body ILIKE '%'new'%' ) LIMIT $2`

Reply:
But what we need is `SELECT "notes".* FROM "notes" WHERE "notes"."user_id" = $1 AND (title ILIKE '%new%' OR body ILIKE '%new%' )`.

Reply:
Ah ok. Makes sense then 👍🏼